docs/en/sql-reference/sql-statements/data-definition/ALTER_TABLE.md

```diff
@@ -421,9 +421,11 @@ Note:
 - All columns in the index must be written.
 - The value column is listed after the key column.
 
-#### Modify columns of the sort key in a Primary Key table
+#### Modify the sort key
 
-<!--Supported Versions-->
+Since v3.3, the sort keys for the Duplicate Key tables, Aggregate tables, and Unique Key tables can be modified. Since v3.0, the sort keys for the Primary Key tables can be modified.
```

Comment:
Since v3.0, the sort keys for the Primary Key tables can be modified. v3.3 extends this support to Duplicate Key tables, Aggregate tables, and Unique Key tables.